msvc2017/Common/Common.vcxproj

```diff
@@ -5,20 +5,24 @@
       <Configuration>Debug</Configuration>
       <Platform>Win32</Platform>
     </ProjectConfiguration>
-    <ProjectConfiguration Include="Release-XP|Win32">
-      <Configuration>Release-XP</Configuration>
-      <Platform>Win32</Platform>
+    <ProjectConfiguration Include="Debug|x64">
+      <Configuration>Debug</Configuration>
+      <Platform>x64</Platform>
     </ProjectConfiguration>
     <ProjectConfiguration Include="Release|Win32">
       <Configuration>Release</Configuration>
       <Platform>Win32</Platform>
     </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|x64">
+      <Configuration>Release</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
   </ItemGroup>
   <PropertyGroup Label="Globals">
     <ProjectGuid>{BEDF460A-EAE9-4E20-AFB2-2C8434051150}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>Common</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.15063.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>8.1</WindowsTargetPlatformVersion>
```

Comment:
@geoffthemedio, does this mean we're raising the minimum required Windows version to 8.1? Not just Win7?

Reply:
Because I didn't have an option for Win7.... just various 10.0.X and 8.1. I'm not actually sure what this setting does though since the old project has this set to `10.0.15063.0` but it still runs on 8.1 for me. But I seemed to need to change it to get it to run, although I may have confused the problem with the GLEW library in the SDK issue.

Reply:
Ok, I see. Well, in that case we don't have much of a choice... I just wonder, because we've been able to support XP up until now, and suddenly we can't support Win7 anymore? Seems weird...